nym-node-status-agent/run.sh

```diff
@@ -1,25 +1,52 @@
 #!/bin/bash
 
 set -eu
+export ENVIRONMENT=${ENVIRONMENT:-"sandbox"}
 
-environment="qa"
-
-source ../envs/${environment}.env
-
-export RUST_LOG="debug"
+probe_git_ref="nym-vpn-core-v1.0.0-rc.6"
 
 crate_root=$(dirname $(realpath "$0"))
-gateway_probe_src=$(dirname $(dirname "$crate_root"))/nym-vpn-client/nym-vpn-core
+monorepo_root=$(dirname "${crate_root}")
+echo "Expecting nym-vpn-client repo at a sibling level of nym monorepo dir"
+gateway_probe_src=$(dirname "${monorepo_root}")/nym-vpn-client/nym-vpn-core
 echo "gateway_probe_src=$gateway_probe_src"
 echo "crate_root=$crate_root"
 
+set -a
+source "${monorepo_root}/envs/${ENVIRONMENT}.env"
+set +a
+
+export RUST_LOG="debug"
+export NODE_STATUS_AGENT_SERVER_ADDRESS="http://127.0.0.1"
+export NODE_STATUS_AGENT_SERVER_PORT="8000"
 export NODE_STATUS_AGENT_PROBE_PATH="$crate_root/nym-gateway-probe"
 
+# each key is required for a separate agent when running in parallel: their
+# public counterparts need to be registered with NS API
+private_keys=("BjyC9SsHAZUzPRkQR4sPTvVrp4GgaquTh5YfSJksvvWT"
+    "4RqSKydrEuyGF8Xtwyauvja62SAjqxFPYQzW2neZdkQD"
```

Comment:
what are those keys? are those for some testnet? whatever they are, i dont think they should be hardcoded in a script...

Reply:
exclusively for testing